src/stitches.config.ts

```diff
@@ -0,0 +1,301 @@
+import { createStitches } from '@stitches/react';
+import type * as Stitches from '@stitches/react';
+export type { VariantProps } from '@stitches/react';
+
+export const MediaQueryKeys = {
+  xs: '(max-width: 520px)',
+  sm: '(max-width: 900px)',
+  md: '(max-width: 1200px)',
+  lg: '(max-width: 1800px)',
+} as const;
+
+export const {
+  styled,
+  css,
+  globalCss,
+  keyframes,
+  getCssText,
+  theme,
+  createTheme,
+  config,
+} = createStitches({
+  theme: {
+    colors: {
+      white: '#fff',
+      black: '#000',
+      // red
+      red200: '#F5E4E4',
+      red300: '#E6BCBC',
+      red400: '#EF7376',
+      red500: '#6f3638',
+      // gray
+      gray400: '#B5BBBD',
+
+      lightBackground: '#E1E1E1',
+      subtleGray: '#F5F5F5',
+      almostWhite: '#F6F7F8',
+      third: '#EAEAEB',
+      gray: '#EFF3F4',
+
+      lightBorder: '#DEDEDE',
+      lightGray: '#C3CDCF',
+      mediumGray: '#A7B0B4',
+      lightText: '#859296',
+      border: '#939EA1',
+      placeholder: '#99A2A5',
+      placeholderDark: '#7C898D',
+
+      primary: '#5E6F74',
+      text: '#516369',
+      secondary: '#56606f',
+
+      background: '#DFE7E8',
+      pacificBlue: '#00ABBF',
+      selected: '#31A5AC',
+      lightBlue: '#84C7CA',
+      darkBlue: '#5D9C9F',
+    },
+    space: {
+      px: '1px',
+      '2xs': '0.125rem', // 2px
+      xs: '0.25rem', // 4px
+      '1sm': '0.375rem', // 6px
+      sm: '0.5rem', // 8px
+      sm1: '0.625rem',
+      sm2: '0.75rem', // 8px
+      md: '1rem', // 16px
+      lg: '1.5rem', // 24px
+      xl: '2rem', // 32px
+      xl2: '2.25rem', // 32px
+      '1xl': '2.5rem', // 40px
+      '2xl': '3rem', // 48px
+      '3xl': '4rem', // 64px
+      '5xl': '5rem', // 80px
+
+      // non standard
+    },
+    sizes: {
+      max: 'max-content',
+      min: 'min-content',
+      full: '100%',
+      px: '1px',
+      '2xs': '0.125rem', // 2px
```

Comment:
these should be the same, like `scale = { px: '1px' }` and then `sizes: { ...scale }` and `space: { ...scale }`

Reply:
agreed

Reply:
It was in that way, some space are shared in boths